cloudigrade/api/authentication.py

```diff
@@ -13,7 +13,7 @@
 logger = logging.getLogger(__name__)
 
 
-def parse_requests_header(request):
+def parse_requests_header(self, request):
```

Comment:
Actually, on second look, since we really only use the `.allow_internal_fake_identity_header` attribute of the new object being passed in, maybe we should just be more precise like:

```suggestion
def parse_requests_header(request, allow_internal_fake_identity_header=False):
```

…and change the caller to send only that attribute, not its whole self.